app/models/api/config.rb

```diff
@@ -0,0 +1,17 @@
+class Api::Config
```

Comment:
LOL, I missed this comment after I pushed. I actually did this exact thing, but then changed it to `.new(hash)` :laughing: